### DISCUSSION TITLE
Adapter for sending/emitting data into cQube

### DISCUSSION THREAD

**Discussion by surendrasinghs** [Sunbird-cQube/community#764] [General]
@sukhpreetsamagra @ChakshuGautam @ananya-jain1 I believe, external systems need to emit data into cQube using adapters. What is the proposed approach for writing an adapter? What are some of the assumptions about the external systems that help us design the adapter?

Can you share the complete ingestion process (emit from source, configure data set, frequency, and monitor ingestion on regular basis)?

@karsood @ikpreet-singh

**Comment by ananya-jain1** [comment]
Hi Suren, we have documented and updated all approaches on the cQube microsite [here](https://sunbird.gitbook.io/sunbird-cqube/). You can have a look there.
